rtl/eclrtl/rtlfield.cpp

```diff
@@ -4340,7 +4340,7 @@ size32_t RtlBitfieldTypeInfo::buildInt(ARowBuilder &builder, size32_t offset, co
     value &= ~(mask << shift);
     value |= ((val << shift) & mask);
 
-    rtlWriteInt(cur, val, getBitfieldIntSize());
+    rtlWriteInt(cur, value, getBitfieldIntSize());
```

Comment:
This looks like it's fixing a bug, not just suppressing a compiler error. Should possibly be a separate PR therefore.

Reply:
Moved to:  https://github.com/hpcc-systems/HPCC-Platform/pull/16295